Makefile.am

```diff
@@ -1199,6 +1204,7 @@ cups_browsed_LDADD = \
 	$(CUPS_LIBS) \
 	$(AVAHI_LIBS) \
 	$(AVAHI_GLIB_LIBS) \
+	$(EXIF_LIBS) \
 	$(GLIB_LIBS) \
 	$(GIO_LIBS) \
 	$(GIO_UNIX_LIBS)
```

Comment:
Also here the additions for libexif are not needed ...

Reply:
Sorry Till, the changes you reviewed are old ones, I am working on a separate branch to resolve these issues and push the same and raise a pr soon